app/bettermail.go

```diff
@@ -24,33 +26,62 @@ func init() {
 	http.HandleFunc("/hook-test-harness", hookTestHarnessHandler)
 	http.HandleFunc("/test-mail-send", testMailSendHandler)
 	http.HandleFunc("/_ah/bounce", bounceHandler)
+	http.HandleFunc("/test-subject", testSubjectHandler)
+}
+
+type EmailThread struct {
+	CommitSHA string
+	Subject string
+}
+
+func createThread(sha string, subject string, c context.Context) {
+	thread := EmailThread {
+	    CommitSHA: sha,
+	    Subject: subject,
+	}
+	key := datastore.NewIncompleteKey(c, "EmailThread", nil)
```

Comment:
See https://github.com/mihaip/slack-archive/blob/master/app/account.go for an example.